extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -165,6 +173,7 @@ export class AmpAnalytics extends AMP.BaseElement {
       this.iniPromise_.then(() => {
         this.transport_.maybeInitIframeTransport(
             this.getAmpDoc().win, this.element);
+        this.linkerManager_.enableFormSupport();
```

Comment:
Looks like you fail to check `'linker-form'` experiment or wait for `allLinkerPromises_ ` here.

Reply:
Good catch. Added experiment protection. The `allLinkersPromises_` part is actually leftover from when we were waiting to scan the page for forms, and is no longer necessary, so it was removed.